schema/rpc.fbs

```diff
@@ -99,8 +99,7 @@ table SteamVRTrackersSetting {
 
 table FilteringSettings {
     type: solarxr_protocol.datatypes.FilteringType;
-    intensity: uint8;
-    ticks: uint8;
+    amount: float32;
```

Comment:
What is the rationale for this backwards-incompatible change? Can it be done in a way that doesn't break anything?

Would anything be broken *in practice* by accepting this change?

Reply:
Also, amount is not descriptive (neither were the original names though). Still, being more descriptive or adding a comment would be good.

Reply:
I’m not too sure what you mean.
Ticks is getting removed since it’s no longer needed.
Intensity is getting renamed to Amount since imo, that is more descriptive (“amount of filtering applied” instead of “intensity of filtering”)

Filtering changed a lot, so GUI will need to be updated regardless…

Reply:
So the GUI (ideally) would work with older versions of the server. Its not a hard requirement, but its something to keep in mind since it is churn in between versions.

Perhaps we should mark the no longer used fields as deprecated for now, and then remove them after the next release of the GUI + Server. Remember that fields in a flatbuffer will take up no space if simply not written to.

You can rename intensity to amount if you want, but you need to:
1. Keep the field in the same position
2. Add a comment better describing this, because intensity and amount are both vague

Reply:
actually, I'm changing my mind. We don't commit to stability yet and all that deprecating the field does is push the problem further down the line - when you inevitably decide there is a new subsequent field, its position will move when we delete the deprecated field.

Reply:
nvm, resolving. break all the things